test/extractors/mfcc.js

```diff
@@ -29,8 +29,11 @@ describe('mfcc', function () {
       -0.5572731218776171,
       -0.5245475651926765,
       -0.6215721667512493
-    ]
-    );
+    ];
+    
+    for (var index in expectedValues) {
+      assert.approximately(en[index], expectedValues[index], 0.000000000000001);
```

Comment:
personally I'd use `1e-15` here and in the `spectralFlatness` test, it means people won't have to count the zeroes 😉 just a little nitpick that is